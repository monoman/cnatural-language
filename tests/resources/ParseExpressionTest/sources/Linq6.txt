query = from n in names
            join p in people on n equals p.Name into matching
            select new { Name = n, Count = matching.Count() }