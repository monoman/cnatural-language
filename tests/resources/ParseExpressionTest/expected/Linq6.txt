Assign(Name(query) Assign from n in Name(names)
join p in Name(people) on Name(n) equals (Name(p)).Name into matching
select new{Name=Name(n), Count=(Name(matching)).Count()})